packages/shared/src/components/modals/AccountDetailsModal.tsx

```diff
@@ -28,78 +30,136 @@ export default function AccountDetailsModal({
 
   return (
-    <ResponsiveModal
-      className={classNames(className, styles.accountDetailsModal)}
-      {...props}
-    >
-      <header className="flex fixed responsiveModalBreakpoint:sticky top-0 left-0 z-3 justify-between items-center px-4 w-full h-12 border-b bg-theme-bg-secondary border-theme-divider-tertiary">
-        <Button
-          title="Close"
-          onClick={props.onRequestClose}
-          icon={<XIcon />}
-          className="btn-tertiary"
-        />
-        <Button
-          buttonSize="small"
-          type="submit"
-          disabled={disableSubmit}
-          form="profileForm"
-          className="btn-primary"
-        >
-          Save changes
-        </Button>
-      </header>
-      <div className="px-4 pt-6 pb-4">
-        <ProfileHeading>Account Details</ProfileHeading>
-        <EditImageWithJoinedDate user={user} />
-        <ProfileForm
-          id="profileForm"
-          setDisableSubmit={setDisableSubmit}
-          onSuccessfulSubmit={() => props.onRequestClose(null)}
-          mode="update"
-        />
-      </div>
-      <footer className="flex flex-col px-4 pb-10 -my-1">
-        <FooterLink href={faq} target="_blank" rel="noopener">
-          FAQ
-        </FooterLink>
-        <FooterLink href={requestFeature} target="_blank" rel="noopener">
-          Request a feature
-        </FooterLink>
-        <FooterLink href={reportIssue} target="_blank" rel="noopener">
-          Report an issue
-        </FooterLink>
-        <FooterLink href={privacyPolicy} target="_blank" rel="noopener">
-          Privacy policy
-        </FooterLink>
-        <FooterLink href={cookiePolicy} target="_blank" rel="noopener">
-          Cookie policy
-        </FooterLink>
-        <FooterLink href={termsOfService} target="_blank" rel="noopener">
-          Terms of service
-        </FooterLink>
-        {user.premium && (
-          <FooterLink
-            href="mailto:support@daily.dev?subject=Cancel my premium subscription"
-            target="_blank"
-            rel="noopener"
+    <>
+      <ResponsiveModal
+        className={classNames(className, styles.accountDetailsModal)}
+        {...props}
+      >
+        <header className="flex fixed responsiveModalBreakpoint:sticky top-0 left-0 z-3 justify-between items-center px-4 w-full h-12 border-b bg-theme-bg-secondary border-theme-divider-tertiary">
+          <Button
+            title="Close"
+            onClick={props.onRequestClose}
+            icon={<XIcon />}
+            className="btn-tertiary"
+          />
+          <Button
+            buttonSize="small"
+            type="submit"
+            disabled={disableSubmit}
+            form="profileForm"
+            className="btn-primary"
           >
-            Cancel subscription
+            Save changes
+          </Button>
+        </header>
+        <div className="responsiveModalBreakpoint:px-10 pt-6 pb-4 px-6z">
```

Comment:
I think there's a typo mistake.
```suggestion
        <div className="responsiveModalBreakpoint:px-10 pt-6 pb-4 px-6">
```

Reply:
I think it would be better to use `mobileL` as the `responsiveModalBreakpoint` might serve a different purpose and be changed in the future. Also just to be specific with @tsahimatsliah's request of using `Mobile L`

```suggestion
        <div className="mobileL:px-10 pt-6 pb-4 px-6z">
```